lib/builder-webpack5/src/presets/module-federation.ts

```diff
@@ -0,0 +1,113 @@
+import type { Configuration, EntryObject, WebpackPluginFunction } from 'webpack';
+import { logger } from '@storybook/node-logger';
+import WebpackVirtualModules from 'webpack-virtual-modules';
+
+interface VirtualModules {
+  [key: string]: string;
+}
+
+interface FinalEntries {
+  entries: EntryObject;
+  virtualModules: VirtualModules;
+}
+
+const sampleEntry: EntryObject = {};
+
+type EntrtDesc = typeof sampleEntry.sample;
```

Comment:
optional - For consistency, I would expand the abbreviation.

Reply:
Whoops, this is a typo from rapid experimenting, I'll update the name.

Reply:
Done